id-policy.md

```diff
@@ -0,0 +1,231 @@
+ - Each OBO id is assigned to a only single term within the set of all OBO ontologies.
+ - There is a 1:1 mapping of OBO ids to Foundry-compliant URIs
+
+#### Format of OBO format ids
+
+See [OBO syntax](http://owlcollab.github.io/oboformat/doc/obo-syntax.html) for details.
+
+All OBO term IDs are CURIEs (prefixed identifiers) of the form `IDSpace:LocalId`
+
+Note that internally within an OBO Format document, relation IDs may
+be short form such as `part_of`, but these are mapped to CURIEs via the xref field.
+
+#### Format of Foundry-compliant URIs
+
+`FOUNDRY_OBO_URI ::= "http://purl.obolibrary.org/obo/" IDSPACE "_" LOCALID`
+
+#### Mapping of OWL ids to OBO format ids
+
+The recommended way is to use the [OBO JSON LD context](https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/registry/context.jsonld)
+
+This provides mappings between the OBO prefix and a base URL. See the [JSON-LD spec](https://json-ld.org/spec/latest/json-ld/) for a full specification of the expansion/contraction mechanism.
+
+CURIE to URI:
+
+`s@((**[A-Za-z_]***):(d+))@http://purl.obolibrary.org/obo/$1_$2@`
+
+URI to CURIE
+
+`s@http://purl.obolibrary.org/obo/(**[A-Za-z_]***)_(d+)@$1:$2@`
+
+
+Using these rules the OBO id**GO:0050918** is mapped to
+ the Foundry-compliant URI
+**http://purl.obolibrary.org/obo/GO_0050918** 
+
+## Policy for OBO Foundry ontologies
+
+All OBO ontologies should use PURLs that have a base URL within their allocated space (in CURIE form these would be prefixed with their ID space)
+
+
+## Response to Web requests for OBO URIs
+
+It is expected that the Foundry-compliant URIs behave, on the web,
+usefully. It will be the role of the OBO Foundry to supply generic
+software for responding to requests at URIs that identify OBO terms.
+
+
+We borrow the criteria from the Shared Name Initiative
+(http://sharedname.org/)as a base line. The OBO Foundry may issue
+further recommendations if experience shows them to be considered
+generally useful.
+
+
+
+1.  It must be clearly stated what the intended referent of each URI is
+    supposed to be, i.e. that the URI denotes some particular record
+    from some particular database.
+
+2.  Information about the URI and its referent, including such a
+    statement, must be made available, and in order to leverage existing
+    protocol stacks, it must be obtainable via HTTP. (We will call such
+    information "URI documentation".)
+
+3.  URI documentation must be provided in RDF.
+
+4.  Provision of URI documentation must be an ongoing concern. The
+    ability to provide it may have to outlive the original ontology
+    developer's group or creator.
+
+5.  The provider of the URI documentation must be responsive to
+    community needs, such as the need to have mistakes fixed in a timely
+    manner.
+
+6.  URI documentation must be open so that it can be replicated and
+    reused.
+
```

Comment:
I think the above are a bit verbose and not specific. Eg 1. refers to a database, whereas in most cases there are no database for ontologies.